### ISSUE TITLE
Update iNaturalist

### ISSUE THREAD

**Issue by sadeghim** [AtlasOfLivingAustralia/data-management#175]
Develop an automated feed for inaturalist.

**Comment by djtfmartin** [comment]
I thought we already have this. See jenkins job:

http://ala-mccartney.it.csiro.au/job/iNaturalist%20weekly/

**Comment by sadeghim** [comment]
Yes, you are right.
There is a job for that but the data that we have are 10249 records (http://biocache.ala.org.au/occurrences/search?q=data_resource_uid:dr1411) which fetches a zip file prepared specifically for ALA and resides on their side.

But the last run that I had on their website through APIs contains 29551 records (http://ala-dury.it.csiro.au/ala-hub/occurrences/search?q=data_resource_uid:dr1411). 

And I guess they have not updates the process of preparing this file for us or it has a filter on it serving only one third of the records to us.

**Comment by sadeghim** [comment]
Now we have more than 30K records from them:
http://biocache.ala.org.au/occurrences/search?q=data_resource_uid:dr1411#tab_recordImages

The job is automated and will fetch the records daily.

**Comment by ansell** [comment]
The feed may have broken recently, so reopening to investigate the current status.

**Comment by ansell** [comment]
The overnight run took 15 hours.

**Comment by peggynewman** [comment]
The API has an updated_since parameter so we should be able to pull deltas from the API. The previous load was 300K+ records and daily deltas look to be around 1000 records. @sadeghim as we discussed and I've checked with @M-Nicholls - ideally we would get the deltas daily with the understanding that the SOLR index only runs twice/week so records would only appear then. We can rerun the bulk load if any issues arise.
